src/Uart/Uart.c

```diff
@@ -53,6 +53,38 @@ getAddressBase(Uart_Id id)
     }
 }
 
+static inline bool
+setBaudRate(const Uart* const uart, const Uart_BaudRate baud)
+{
+    uint32_t baudValue = 0;
+    switch(baud) {
+        case 300:
```

Comment:
Uart_BaudRate is an enum... why use raw values?

Reply:
Changed